macrosynergy/download/dataquery.py

```diff
@@ -743,6 +743,8 @@ def get_ts_expression(
         c_delay = self.delay_compute(len(expressions))
         results = None
 
+        print(datetime.utcnow().strftime("%Y-%m-%d %H:%M:%S.%f")[:-3] + " UTC")
```

Comment:
Alternative: 
```python
print(datetime.utcnow().isoformat())
```

Reply:
Changed to `print(datetime.utcnow().isoformat(), " UTC")` to avoid confusion